### PR TITLE
OPSEXP-2203 Restart affected services by certificate refresh for mtls

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-2203

### CODE REVIEW

File: roles/repository/tasks/restart.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- name: Restart {{ svc }}
+  become: true
+  throttle: 1
+  vars:
+    svc: alfresco-content{% if groups.repository | length > 1 %}-monitored-startup{% endif %}
+  ansible.builtin.service:
+    name: "{{ svc }}"
+    state: restarted
+    daemon_reload: true
```
Comment (gionn) [inline roles/repository/tasks/restart.yml:9]: not really needed?

```suggestion
```

File: roles/java/tasks/keystores.yml [hunk 2]
```diff
@@ -28,6 +28,7 @@
         mode: "0400"
         dest: "{{ download_location }}"
       register: certcp
+      notify: restart-java-services
```
Comment (alxgomz) [inline roles/java/tasks/keystores.yml:31]: I thikn this handler would be better placed in the task "Import certificates to a keystore" & "Trust certificates CA"

File: roles/java/tasks/keystores.yml [hunk 3]
```diff
@@ -164,3 +165,6 @@
           -alias {{ item.name }}
       tags:
         - molecule-idempotence-notest
+
+- name: Force all notified handlers to run
+  ansible.builtin.meta: flush_handlers
```
Comment (alxgomz) [inline roles/java/tasks/keystores.yml:170]: Why do we need that? Ins't it OK to run at the end of the play?

File: roles/repository/tasks/restart.yml (hunk 1 above)
Comment (alxgomz) [reply roles/repository/tasks/restart.yml:9]: should not indeed but better to keep it i case notrification order  changes or we one day need to include just that file

**Review by alxgomz — COMMENTED** [review]
just wondering about the flush_handler usage

File: roles/java/tasks/keystores.yml (hunk 3 above)
Comment (alxgomz) [reply roles/java/tasks/keystores.yml:171]: make sense indeed

**Review by alxgomz — APPROVED** [review]

**Pull request merged**